src/status_im/signals/core.cljs

```diff
@@ -12,14 +12,23 @@
 (fx/defn status-node-started
   [{db :db :as cofx}]
   (fx/merge cofx
-            {:db (assoc db :status-node-started? true)}
-            #(when (:password (accounts.db/credentials cofx))
+            (merge
+             {:db (assoc db
+                         :status-node-state    :started
+                         :dispatch-after-start nil)}
+             (when (:dispatch-after-start db)
+               {:dispatch-n (:dispatch-after-start db)}))
+            #(when (and (nil? (:dispatch-after-start db)) ;; node is not going 
```

Comment:
at some point in time we agreed to not add comments to the end of the line, but place them above form, can't find a link...